apps/ecommerce/frontend/src/components/Dialog/Dialog.styles.ts

```diff
@@ -0,0 +1,47 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+
+const HEADER_HEIGHT = 114;
```

Comment:
I took much of this css from the existing ecom base app, I imagine we will update this css as we continue to build out the search/filter/selection functionality in the dialog.